_includes/tutorials/message-ordering/kafka/code/tutorial-steps/test/dump-log-segments-0.sh

```diff
@@ -0,0 +1,4 @@
+docker-compose exec broker kafka-run-class kafka.tools.DumpLogSegments \
```

Comment:
Instead of `kafka-run-class kafka.tools.DumpLogSegments`, there's a script `kafka-dump-log` which can be used.

Reply:
@mikebin I was not aware of that, thank you!  I will validate and use `kafka-dump-log`.